book/src/snapshot-restore-feature.md

```diff
@@ -2,10 +2,11 @@
 
 ## Status
 
-Status | Min K8s Version | Max K8s Version | external-snapshotter Version | external-provisioner Version
---|--|--|--|--
-Alpha | 1.12 | 1.12 | 0.4 | 0.4
-Alpha | 1.13 | - | 1.0+ | 1.0+
+Status | Min K8s Version | Max K8s Version | snapshot-controller Version | CSI external-snapshotter sidecar Version | external-provisioner Version
+--|--|--|--|--|--
+Alpha | 1.12 | 1.12 | | 0.4+ | 0.4+
+Alpha | 1.13 | 1.16 | | 1.0+ | 1.0+
```

Comment:
These versions have upper bounds now

Reply:
Should we just remove this table and refer to the tables in external-snapshotter.md and snapshot-controller.md?  I found it very confusing to maintain tables with duplicate information in different files.

Reply:
Ok, we probably still need this as this has everything together including the external-provisioner.